db/migrate/20190827162435_create_targets.rb

```diff
@@ -0,0 +1,16 @@
+class CreateTargets < ActiveRecord::Migration[5.2]
+  def change
+    create_table :targets do |t|
+      t.belongs_to  :topic, null: false
```

Comment:
I'd use `t.references` for both cases. Also this creates the foreign key
[check this](https://stackoverflow.com/questions/16309742/t-references-in-the-migration-vs-belongs-to-in-the-model/16309795)